packages/utils/src/types/ticket.spec.ts

```diff
@@ -1,17 +1,20 @@
 import assert from 'assert'
 import { expect } from 'chai'
-import { stringToU8a, SIGNATURE_LENGTH } from '@hoprnet/hopr-utils'
+import { stringToU8a, SIGNATURE_LENGTH } from '..'
 import { ethers } from 'ethers'
 import { Address, Ticket, Hash, Balance, PublicKey, Signature, UINT256 } from '.'
-import * as fixtures from '../fixtures'
 import BN from 'bn.js'
 import { randomBytes } from 'crypto'
+import { Wallet } from 'ethers'
+
+const ACCOUNT_A = new Wallet('0x4471496ef88d9a7d86a92b7676f3c8871a60792a37fae6fc3abc347c3aa3b16b')
+const ACCOUNT_B = new Wallet('0x18a664889e28a432495758f0522b53b2f04a35f810b78c6ea01db305141bcba2')
```

Comment:
Is this the only place we use this? If that's the case I'm fine having this written directly, otherwise, I do not see any issue having to import the fixtures.

Reply:
the reason I don't want to import is to avoid utils relying on core-ethereum.